www/index.html

```diff
@@ -440,7 +441,9 @@ <h3 id="other" data-i18n="about-other-platforms">Other platforms/versions</h3>
                     <br/>
                     <br/>
                 </div>
-
+                <div id='library' style="display: none; color: white;">
+                    <iframe frameborder="0" width="100%" id="libraryIframe"></iframe>
```

Comment:
I think the ID of this iframe should be `libraryContent` to mirror the other iframe's ID `articleContent`.